host/xtest/regression_1000.c

```diff
@@ -419,7 +420,7 @@ static void xtest_tee_test_1003(ADBG_Case_t *c)
 			    (double)(repeat * num_readers);
 
 	Do_ADBG_Log("    Number of parallel threads: %zu (%zu writers and %zu readers)",
-		    num_threads, num_writers, num_readers);
+		    TEST_1003_THREAD_COUNT, num_writers, num_readers);
```

Comment:
`TEST_1003_THREAD_COUNT` should be formatted as an `int` (`%d`) instead of `size_t`

Reply:
ok, thanks.